source/transactions-convenient-api/tests/README.rst

```diff
@@ -196,8 +196,8 @@ should be checked:
    the retry timeout has been exceeded, ``withTransaction`` should propagate the
    error to its caller.
  * If committing raises an error with the UnknownTransactionCommitResult label,
-   the error is not a write concern timeout, and the retry timeout has been
-   exceeded, ``withTransaction`` should propagate the error to its caller.
+   and the retry timeout has been exceeded, ``withTransaction`` should
+   propagate the error to its caller.
```

Comment:
If you're revising a prose test, I think this warrants a changelog entry. This should also be highlighted in the final description of DRIVERS-555 so languages know to consider this change and adjust their prose tests accordingly.